main.lua

```diff
@@ -622,6 +622,16 @@ function LSPClient:receiveMessage(text)
     elseif decodedMsg.method then
         self:handleNotification(decodedMsg)
     else
+        if decodedMsg.result == nil then
+            local request = self.sentRequests[decodedMsg.id]
+            self.sentRequests[decodedMsg.id] = nil
+            if request then
+                display_info(string.format("No result for %s", request))
+            else
+                display_error("No result")
```

Comment:
I don't think this error is useful. When there was no sent request displaying a "No result" error is misleading.